db_master_deploy/deploy.sh

```diff
@@ -361,8 +392,6 @@ for source_object in "${array_source[@]}"; do
     array=()
 done
 
-# check for lockfile, if there is one exit script, lock file is created by import_databases.sh
-(ls ${lockfile} &> /dev/null) && { echo "lockfile found: ${lockfile} '$(cat ${lockfile})'" >&2; exit 1; }
```

Comment:
the lock file was used temporary during the import of databases. it is not needed anymore.